docsrv/fetcher_test.go

```diff
@@ -12,8 +13,9 @@ const (
 )
 
 func TestReleases(t *testing.T) {
+	apiKey := os.Getenv("GITHUB_API_KEY")
```

Comment:
Should we use `src-d/framework/configurable`?
(and kind beyond this PR: should we release `configurable` as an independent library? it looks like something that could be already considered stable and versioned as `v1`)